modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,504 @@
+
+Total Requests::
+Total number of requests to the listed entities
+
+Error Rate:: 
+For entities with request errors, the percentage equal to the total number of request errors divided by the total number of requests
++
+For example, assume that 1 out of 5 total requests to _API A_ produces an error and that 2 requests out of 5 to _App B_ produces an error. The error rate is 3/10, or 30%. 
+
+//
+// H4: Performance Trends
+[[performance-trends-overview]]
+==== Performance Trends
+
+Monitor high-level performance over time across your entities. 
+include::partial$include-entities.adoc[tags="entity-definition"]
+
+include::partial$include-insights-charts.adoc[tags="data-scope"]
+
+Entities with Slow Response Times::
+Response times, in milliseconds, of the entities with the slowest response times
++
+The metric uses percentile values for response times (calculated from highest to lowest times) to identify entities with the slowest responses. You can select the *p99* (default), *p95*, *p90*, or *p75*.
+
+Error Volume:: 
+Number of errors received by the entities over time
+
+* For API implementations, this metric counts HTTP error codes (for example, `404` or `502`). 
+* For Mule apps, this metric counts error response codes (for example, `FAILED`).
+
+Request Volume:: 
+Number requests over time for the entities 
+
+//
+//H3: ENTITIES LIST
+[[entities-list]]
+=== Entities List
+
+Check key metrics on individual Mule apps and APIs (_entity_), and click an entity's name to get many detailed charts on the entity. 
+include::partial$include-entities.adoc[tags="entity-definition"]
+
+include::partial$include-insights-charts.adoc[tags="data-scope"] The entities belong to the selected <<change-bg, business group>>.
+
+*Entities List* provides these metrics:
+
+Entity Name:: 
+Name of the entity
+
+//NOTE: changed value Api -> API
+Entity Type:: 
+Type of entity, such as *API* or *Application* (Mule app)
+
+Request Volume::
+Total number of requests to this entity
+
+Response Time (p99):: 
+Response time, in milliseconds, for the 99th percentile of requests to this entity
++
+This response time is greater than 99% of all response times for this entity.
+
+Error Rate:: 
+Percentage of requests to this entity that resulted in errors 
+
+Deployment Type:: 
+Deployment target of this entity, such as *CloudHub 2.0* or *Runtime Fabric* 
+
+//
+//H3: ENTITY METRICS for APPS (ONLY)
+[[entity-metrics-apps]]
+=== Application Details
+
+Anypoint Monitoring provides details on individual Mule apps deployed to a supported target.
+
+* <<entity-metrics-app-header>>
+* <<entity-metrics-app-overview>>
+* <<entity-metrics-app-requests>>
+* <<entity-metrics-app-errors>>
+* <<entity-metrics-app-jvm>>
+
+include::partial$include-entities.adoc[tags="entity-definition"]
+
+[[entity-metrics-app-header]]
+==== Mule Application Metrics
+
+High-level metrics are available in Application Details dashboards for individual Mule apps. 
```

Comment:
Should Application Details be bold (UI element) or lowercase (generic)?

Reply:
hmm, need to double check. Don't remember why capitalized. 
TODO; @dukesphere